mist-core/src/main/java/edu/snu/mist/core/parameters/MasterHostname.java

```diff
@@ -21,6 +21,6 @@
 /**
  * The Mist master host name.
  */
-@NamedParameter(doc = "The mist master host name")
+@NamedParameter(doc = "The mist master host name", default_value = "127.0.0.1")
```

Comment:
Could you please remove the default value? It would be good to remove it for debugging.